source/Octopus.Server.Client/Repositories/Async/BasicRepository.cs

```diff
@@ -253,6 +254,26 @@ public async Task<List<TResource>> GetAll(CancellationToken cancellationToken)
             return await Client.Get<List<TResource>>(link, parameters, cancellationToken).ConfigureAwait(false);
         }
 
+        public Task<List<TResource>> FindByPartialName(string partialName, string path, object pathParameters, CancellationToken cancellationToken)
+        {
+            ThrowIfServerVersionIsNotCompatible(cancellationToken).ConfigureAwait(false);
```

Comment:
It just occurred to me that we need to `await` this.

Looks like a bug that we're not awaiting this in a bunch of other methods in this class too 😞 You don't have to fix them now (besides this one) unless you want to, if you don't PET will handle it.

Reply:
Yeah no worries, I'll fix up my usage of it. Fixing the others is probably best to leave for a standalone PR